parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -643,68 +643,73 @@ logs_to_events() {
     fi
 }
 
-should_skip_nvidia_drivers() {
+get_imds_metadata_body() {
     set -x
     body=$(curl -fsSL -H "Metadata: true" --noproxy "*" "http://169.254.169.254/metadata/instance?api-version=2021-02-01")
     ret=$?
     if [ "$ret" -ne 0 ]; then
+      export IMDS_BODY=""
```

Comment:
as such, this should be named something like `IMDS_INSTANCE_METADATA`

Reply:
updated